michaelmsinclair-toronto.md

```diff
@@ -0,0 +1,8 @@
+# Toronto, Ontario, Canada
+
+### Restaurants
```

Comment:
@michaelmsinclair I ❤️ Chinese food. Do you have any Chinese restaurant recommendations?